doc/src/properties/residue.toml

```diff
@@ -109,8 +113,10 @@ Examples include ``extended``, ``alpha helix``, ``pi helix``, ``turn`` and
 ``coil``. This property is not set for undefined secondary structures and is not
 used for writing."""
 
-[resname]
+[segname]
 type = "string"
 PDB = """``PDB`` files used with CHARM, NAMD and a few other software package
 can contain a non-standard *segment* name associated with a residue. If set when
 writting a frame, this property is written to the file."""
+
+PSF = """ TODO """
```

Comment:
@ShoubhikRaj what would be a good description of the segname property? This will be rendered as part of the documentation (http://chemfiles.org/chemfiles/latest/properties.html#residue-properties)

Reply:
@Luthaf No idea sorry. What I do know is that the segName is used by VMD for some particular purposes. For example, if you want to solvate a system, the water model files supplied has to have the segName "QQQ" for the water molecules otherwise the VMD won't be able to determine which one is the solvent. (Using the `solvate` plugin)